content/posts/saud-ansari/sae-baja/index.md

```diff
@@ -0,0 +1,46 @@
+---
+template: "post"
+slug: "sae-baja"
+
+title: "Sae Baja"
+author: "Saud Ansari"
+
+cover: "./banner.jpg"
+images:
+    - banner:
+        url: "https://unsplash.com/photos/7E6QksW4PUY"
+        feature: ./banner.jpg
+        credit:
+            info: "Image taken from unsplash.com"
+            link: https://unsplash.com/@suddenpanic
+
+    - two buggies:
```

Comment:
we would use underscore here not space

Reply:
like this?
info: image_taken_from_unsplash.com

Reply:
yes